shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -105,619 +70,23 @@ public void close() throws IOException {
-          while (in.read(buf) != -1) {
-          }
-        } catch (Exception e) {
-          throw closer.rethrow(e);
-        } finally {
-          closer.close();
-        }
-      }
-      System.out.println(filePath + " loaded");
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-  }
-
-  /**
-   * Persist a file or directory currently stored only in Tachyon to the UnderFileSystem
-   *
-   * @param filePath the TachyonURI path to persist to the UnderFileSystem
-   * @throws IOException when a Tachyon or I/O error occurs
-   */
-  public void persist(TachyonURI filePath) throws IOException {
-    try {
-      TachyonFile fd = mTfs.open(filePath);
-      FileInfo fInfo = mTfs.getInfo(fd);
-      if (fInfo.isFolder) {
-        List<FileInfo> files = mTfs.listStatus(fd);
-        List<String> errorMessages = new ArrayList<String>();
-        for (FileInfo file : files) {
-          TachyonURI newPath = new TachyonURI(file.getPath());
-          try {
-            persist(newPath);
-          } catch (IOException e) {
-            errorMessages.add(e.getMessage());
-          }
-        }
-        if (errorMessages.size() != 0) {
-          throw new IOException(Joiner.on('\n').join(errorMessages));
-        }
-      } else if (fInfo.isIsPersisted()) {
-        System.out.println(filePath + " is already persisted");
-      } else {
-        long size = TachyonFileSystemUtils.persistFile(mTfs, fd, fInfo, mTachyonConf);
-        System.out.println("persisted file " + filePath + " with size " + size);
-      }
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-  }
-
-  /**
-   * Copies a list of files or directories specified by srcFiles from the local filesystem to
-   * dstPath in the Tachyon filesystem space. This method is used when the input path contains
-   * wildcards.
-   *
-   * @param srcFiles The list of files in the local filesystem
-   * @param dstPath The TachyonURI of the destination
-   * @throws IOException
-   */
-  public void copyFromLocalWildcard(List<File> srcFiles, TachyonURI dstPath) throws IOException {
-    try {
-      mTfs.mkdir(dstPath);
-    } catch (FileAlreadyExistsException e) {
-      // it's fine if the directory already exists
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-
-    FileInfo dstFileInfo;
-    try {
-      TachyonFile dstFd = mTfs.open(dstPath);
-      dstFileInfo = mTfs.getInfo(dstFd);
-    } catch (TachyonException e) {
-      throw new IOException(e.getMessage());
-    }
-    if (!dstFileInfo.isFolder) {
-      throw new IOException(
-          ExceptionMessage.DESTINATION_FILE_CANNOT_EXIST_WITH_WILDCARD_SOURCE.getMessage());
-    }
-
-    List<String> errorMessages = new ArrayList<String>();
-    for (File srcFile : srcFiles) {
+  private void loadCommands() {
+    Reflections reflections = new Reflections("tachyon.shell.command");
```

Comment:
Other than using constants, what about getting the package name from this.getClass()?